src/jquery.fancytree.dnd5.js

```diff
@@ -1050,6 +1055,7 @@
 			dragExpand: $.noop, // Callback(targetNode, data), return false to prevent autoExpand
 			dragDrop: $.noop, // Callback(targetNode, data)
 			dragLeave: $.noop, // Callback(targetNode, data)
+			dropMarkerRoot: document.body, // Root Container used for drop marker (could be a shadow root)
```

Comment:
Please rename to 'dropMarkerParent' and move this line up to the related options.